File: Day 3/algorithms/bubble_sort/bubble.py
# Bubble Sort Algorithm example

my_list = [4, 2, 6, 2, 8, 0, 1, 5]
n = len(my_list)

# Do the bubbling up N - 1 times
for i in range(0, n):
    # Last `i` elements are already done, no need to go to the end 
    for j in range(0, n - 1 - i):
        if my_list[j] > my_list[j + 1]:
            my_list[j], my_list[j + 1] = my_list[j + 1], my_list[j]
            
            
print(my_list)

File: Day 3/algorithms/bubble_sort/bubble.js
// * Bubble Sort Algorithm example

let myList = [4, 2, 6, 2, 8, 0, 1, 5];
let n = myList.length;

for (let i = 0; i < n - 1; i++) {
  for (let j = 0; j < n - 1 - i; j++) {
    if (myList[j] > myList[j + 1]) {
      let temp = myList[j];
      myList[j] = myList[j + 1];
      myList[j + 1] = temp;
    }
  }
}

console.log(myList);
